swift/NameNormalizerSwift/NameNormalizer/AuthorNameNormalizer.swift

```diff
@@ -1,3 +1,41 @@
+public enum AuthorNameNormalizerError: Swift.Error {
+    case multipleCommas
+}
+
 public func normalize(name: String) throws -> String {
-    fatalError("normalize not implemented")
+    let suffixParts = try splitSuffix(name)
+    let parts = suffixParts.nameWithoutSuffix.split(separator: " ")
+    if parts.count < 2 {
```

Comment:
the "< 2" suggests 0 and 1... is it possible to trigger the 0 case here?
Also: the parts.count < 2 isn't immediately obvious from a semantic sense, how do you feel about a func to represent the conditional?

Reply:
Yes, this does the 0 case as well.
Yeah, the 2 sitting here all alone is a magic number. Extracting!